api/libproject/devstart.rb

```diff
@@ -1006,6 +1006,52 @@ def update_user_registered_status(cmd_name, args)
   :fn => ->(*args) { update_user_registered_status("update_user_registered_status", args) }
 })
 
+def backfill_gsuite_user_data(cmd_name, *args)
+  ensure_docker cmd_name, args
+
+  op = WbOptionsParser.new(cmd_name, args)
+  op.opts.dry_run = true
+  op.opts.project = TEST_PROJECT
+
+  op.add_typed_option(
+      "--dry_run=[dry_run]",
+      TrueClass,
+      ->(opts, v) { opts.dry_run = v},
+      "When true, print debug lines instead of performing writes. Defaults to true.")
+
+  print "Project is #{op.opts.project}\n"
+
+  # Create a cloud context and apply the DB connection variables to the environment.
+  # These will be read by Gradle and passed as Spring Boot properties to the command-line.
+  gcc = GcloudContextV2.new(op)
+  op.parse.validate
+  gcc.validate()
+
+  if op.opts.dry_run
+    print "DRY RUN -- CHANGES WILL NOT BE PERSISTED\n"
```

Comment:
We've typically used `common.status` for these kinds of notifications. It has a slightly different color I guess to distinguish from stderr from other tools. May want to keep using that here for consistency at least (though I could imagine an argument for switching everything to `print`)

Reply:
Ah I didn't realize – switched, thanks.